### ISSUE TITLE
mysqueezebox.com is obsolete

### ISSUE THREAD

**Issue by TO-92** [kaaholst/android-squeezer#846]
Just a minor note - there is still a "mysqueezebox.com" option on the server connection page. It doesn't hurt, but it is obsolete since Logitech has shut down the server, and could be removed.

**Comment by kaaholst** [comment]
You're right, I haven't got around to this yet. It will be removed in the next version. Thanks for the feedback

**Comment by kaaholst** [comment]
Fixed in 2.3.10 so closing this
